.docker/tutorial-source/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax = docker/dockerfile:1.3
 
-# ghcr.io/ros-planning/moveit2:main-tutorial-${ROS_DISTRO}
+# ghcr.io/ros-planning/moveit2:main-${ROS_DISTRO}-tutorial-source
```

Comment:
Before this change the name was `ghcr.io/ros-planning/moveit2:main-tutorial-humble-tutorial-source`

Reading this I would expect the new name to be `ghcr.io/ros-planning/moveit2:main-humble-tutorial-source-tutorial-source` or am I missing something?

Reply:
this is just a comment about what it is, ROS_DISTRO just expands to the ros distro name

Reply:
I know the code only appends the ROS_DISTRO but where does it append the extra `-tutorial-source` at the end?